earth_enterprise/rpms/build.gradle

```diff
@@ -43,20 +43,6 @@ apply plugin: "nebula.ospackage"
 
 def rpmPlatformString = com.tsciences.os.Platform.rpmPlatformString
 
-def openGeeVersion =
-    new File(project.projectDir, '../src/fusion_version.txt').readLines().
-    collect {
-        // Strip comments:
-        def commentIndex = it.indexOf('#')
-
-        commentIndex < 0 ? it : it.substring(0, commentIndex)        
-    }.
-    collect { it.trim() }. // Trim whitespace.
-    findAll { !it.isEmpty() }. // Skip empty lines.
-    findResult {
-        it // The first string that remains is the version string.
-    }
-
 def stagedInstall_path = '/tmp/fusion_os_install'
 if (new File('build.properties').exists()) {
```

Comment:
what happen is '/tmp/fusion_os_install' does not exist ? should we check and post an error ?

Reply:
This is the error that's returned 
```
FAILURE: Build failed with an exception.

* Where:
Build file '/shared_dev/earthenterprise/earth_enterprise/rpms/build.gradle' line: 79

* What went wrong:
A problem occurred evaluating root project 'rpms'.
> /tmp/fusion_os_install/server/opt/google/gehttpd/htdocs/earth/gee_long_version.txt (No such file or directory)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
```
It doesn't directly say that /tmp/fusion_os_install doesn't exist, but upon investigation, the developer should realize pretty quick as they try to look for the missing file. :) It wouldn't hurt to add a check, but I think it's beyond the scope of this issue as that problem would exist whether or not we're doing the Postgres upgrade. If you think it should be addressed, let's write a follow-up ticket.

Reply:
I'm fine with how it works for now. The error would give the developer a hint of what went wrong.